hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/txns/ethereum/EthereumTransitionLogicTest.java

```diff
@@ -180,15 +180,15 @@ void transitionFailsFastIfNonceDoesntMatch() {
     }
 
     @Test
-    void transitionFailsFastGivenEip2930Txn() {
+    void transitionFailsFastGivenEip2930TxnAndDisabledFeatureFlag() {
```

Comment:
But, where are the _positive_ test(s) for things _working_ if EIP-2930 and feature flag enabled?  Elsewhere?

Reply:
There is an e2e test [here](https://github.com/hashgraph/hedera-services/pull/7696/files#diff-9dfa62e4cc40a03b602c768c3c8c86bd4eb9ca24a5a526732f8f7129302f24c2R308).

Reply:
Is there a good reason why there cannot also be a unit test for the positive case?